template_rpm/packages_fedora.list

```diff
@@ -26,6 +26,7 @@ man-db
 man-pages
 network-manager-applet
 rsync
+selinux-policy-targeted
```

Comment:
This still needs to be conditional on the `selinux` build option. Which means you need to do that in some of the scripts, not the list file.